### PR TITLE
Attempt to package augeas

### PR DESCRIPTION
**Pull request by Leonidas-from-XIV**
This is mostly just freestyled together, let's see if this `opam` file is enough to get augeas installed.

### CODE REVIEW

File: packages/augeas/augeas.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+dev-repo: "git://git.annexia.org/ocaml-augeas.git"
+bug-reports: "augeas-devel@redhat.com"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+  [make "install"]
+]
+depends: [
+  "conf-pkg-config"
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/augeas/augeas.0.6/opam]:
```suggestion
  "ocamlfind" {build}
```

File: packages/augeas/augeas.0.6/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+homepage: "http://augeas.net/"
+dev-repo: "git://git.annexia.org/ocaml-augeas.git"
+bug-reports: "augeas-devel@redhat.com"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+  [make "install"]
+]
+depends: [
+  "conf-pkg-config"
```
Comment (kit-ty-kate) [inline packages/augeas/augeas.0.6/opam]:
```suggestion
  "conf-pkg-config" {build}
```

File: packages/augeas/augeas.0.6/opam [hunk 3]
```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+authors: ["Richard W.M. Jones <rjones@redhat.com>" "Pino Toscano <ptoscano@redhat.com>"]
+maintainer: "Marek Kubica <marek@xivilization.net>"
+homepage: "http://augeas.net/"
+dev-repo: "git://git.annexia.org/ocaml-augeas.git"
+bug-reports: "augeas-devel@redhat.com"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+  [make "install"]
```
Comment (kit-ty-kate) [inline packages/augeas/augeas.0.6/opam]:
```suggestion
]
install: [make "install"]
```

**Comment by Leonidas-from-XIV** [comment]
I have to admit that the OCaml testing CI got much, much better since the last time I submitted a PR - no more spurious failures, all of the failures are 100% my own fault now AND I can rerun random failures if they happen. Very good!

**Comment by avsm** [comment]
Glad to hear it! Thanks for the new package -- you may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**